integration/overrides_exporter_test.go

```diff
@@ -27,23 +27,36 @@ overrides:
 `
 )
 
-func TestOverridesExporterTenantSharding(t *testing.T) {
+func TestOverridesExporterRing(t *testing.T) {
 	limit := 20
 	tests := []struct {
 		name              string
 		flags             map[string]string
 		expectedMetricSum int
 	}{
 		{
-			name: "tenant sharding enabled",
+			name: "ring enabled",
 			flags: map[string]string{
-				"-overrides-exporter.ring.enabled": "true",
+				"-overrides-exporter.ring.enabled":          "true",
+				"-overrides-exporter.ring.heartbeat-period": "1s",
 			},
 			// the limit should only be exported once, i.e. the expected sum of metrics should equal the limit value
 			expectedMetricSum: limit,
 		},
 		{
-			name: "tenant sharding disabled",
+			name: "ring enabled with stability",
+			flags: map[string]string{
+				"-overrides-exporter.ring.enabled":                     "true",
+				"-overrides-exporter.ring.heartbeat-period":            "1s",
+				"-overrides-exporter.ring.heartbeat-timeout":           "2s",
```

Comment:
[question] Is it intention to be so low? If yes, why?

Reply:
No, this is a leftover from the previously implemented approach to waiting at startup. Will remove.